src/Sylius/Behat/Resources/config/suites/api/account/shop_user_account.yml

```diff
@@ -0,0 +1,17 @@
+# This file is part of the Sylius package.
+# (c) Paweł Jędrzejewski
+
+default:
+    suites:
+        api_shop_user_account:
```

Comment:
```suggestion
        api_customer_account:
```

Reply:
api_customer_account is in Resources/config/suites/api/account/customer.yml